### ISSUE TITLE
Unexpected token '.' await pageContext.exports.onHydrationEnd?.(pageContext);

### ISSUE THREAD

**Issue by nestle49** [vikejs/vike#1443]
### Description

![Снимок экрана 2024-01-17 в 13 53 43](https://github.com/vikejs/vike/assets/15983233/f5ed350a-e322-4b21-875e-f35804b1ecd3)

Android 5.5.1 Nexus 7 Chrome 71 (yes, it's problem of old devices)

**Comment by nestle49** [comment]
I would like to support if possible

That is, if we could replace the chain operator in this place and not change anything else, while the old devices could work, then it would definitely be worth it

What do you think about it?

**Comment by brillout** [comment]
I agree. I'm just not sure how much we should prioritize that.

Vike is built by TypeScript. Maybe TypeScript has a setting that transpiles the conditional chain operator away?

**Comment by nestle49** [comment]
I think chain operator setting doesn't exist, at least I didn't find it, so I can't fix it on my side

**Comment by AurelienLourot** [comment]
@nestle49 do you know if this is the only problem you're going to hit on such an old browser? Because maybe even hypothetically if we find a way to remove the chain operator, you'll then hit the next problem out of twenty

**Comment by nestle49** [comment]
> @nestle49 do you know if this is the only problem you're going to hit on such an old browser? Because maybe even hypothetically if we find a way to remove the chain operator, you'll then hit the next problem out of twenty

Yes, I thought about it and I understand that this may turn out to be the case

But this may also be the only problem, in which case it will definitely be worth it

That's why I wanted to discuss this point

**Comment by nestle49** [comment]
I started downgrading chrome versions in target and realized that this is really just the tip of the iceberg (at least for my project), but for others this may not be the case

Still, I’m probably inclined to agree with your point of view that the problem is not worth solving

**Comment by brillout** [comment]
I was thinking of TypeScript's `compilerOptions.target`, does downgrading it get rid of the operator?

Closing, but feel free to continue pushing for this and PR welcome.
